content/blogs/newsletter-102.md

```diff
@@ -0,0 +1,115 @@
+
+The answer is certainly no, the way we handle releases has changed. Back in the day the way ReactOS releases were made were done for the sake of quantity rather than quality. Every new release after 3 months is similar to the last one, with the exceptions of a small number of features and bug fixes added to the release. But overall the net gap between the releases wasn't that big.
+For a while now we have instated a rule that for a new version to reach "Release" status, it has to have a reasonable amount of regressions (no more than 20) and that the stability isn't badly impacted by the introduction of new features or code changes done during the development.
+
+In other words, the ReactOS project is now focusing on providing quality releases, which rules out the 3-months release cadence. This approach wasn't quite feasible for a development team with a size like ours anyway.
+
+It is expected that it will take longer for a new release to be published, although these releases will now be much more substantial, with a much bigger number of improvements.
+If you're impatient to test the latest changes and features you can always try the [nightly builds](https://reactos.org/getbuilds) (also called experimental builds).
+
+## Development status
+In the last year and during the beginning of 2023, the ReactOS developers and contributors alike are working on many parts of the project, the top focused area being the kernel.
+Other areas that aren't kernel related are the applications, specifically our Paint and Notepad programs, the Input Method Editor (IME) as well as other stuff such as the ReactOS testcase infrastructure.
+
+## ReactOS x64 port status
+The x64 port of ReactOS, while still in its infancy, has steadily gotten better in terms of stability. Back then ReactOS could barely boot and if it ever did, it usually resulted in an immediate crash or a BSoD by just doing simple tasks. With the help of Timo Kreuzer (tkreuzer), the x64 port has gotten to a somewhat tolerable state where ReactOS can now operate without big problems.
+
+Although issues still do remain, and compatibility is inferior to the x86 version, more applications and even hardware drivers have started to work.
+Note that the x64 port currently will not run **ANY** x86 applications due to lack of WoW64.
+
+{{< gallery >}}
+{{< figure link="/img/blogs/newsletter-102/vlc.png" src="/img/blogs/newsletter-102/vlc.png" >}}
+{{< figure link="/img/blogs/newsletter-102/emule.png" src="/img/blogs/newsletter-102/emule.png" >}}
+{{< figure link="/img/blogs/newsletter-102/sumatra.png" src="/img/blogs/newsletter-102/sumatra.png" >}}
+{{< figure link="/img/blogs/newsletter-102/mixxxdj.png" src="/img/blogs/newsletter-102/mixxxdj.png" >}}
+{{< /gallery >}}
+
+## WHS test fixes
+The WHS test bot (acronym that stands for Windows Home Server) is a bot that runs Windows Server 2003 where some of its system files are exchanged with ReactOS' system files.
```

Comment:
No, that's wrong. Windows Home Server is just a poor man's Windows Server 2003 (literally, it was 80€ in 2011).
We don't exchange any files by ReactOS files on that machine (such a testbot used to exist, but we're not running it at the moment).